web-app/src/Components/SightForm/Form/index.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
```

Comment:
This file should be something like `Sights/New/Form/index.js` but not `..Form/Form`